apps/balancer-tools/src/app/apr/api/(utils)/filter.ts

```diff
@@ -59,8 +59,8 @@ const conditions: ConditionTypes = {
     value!
       .map((pType) => pType.toLowerCase())
       .includes(pool.type.toLowerCase()),
-  minTvl: (pool, value) => pool.tvl >= value!,
-  maxTvl: (pool, value) => pool.tvl <= value!,
+  minTvl: (pool, value) => pool.tvl >= value! && value! >= 10_000,
+  maxTvl: (pool, value) => pool.tvl <= value! && value! <= 10_000_000_000,
```

Comment:
if you want to keep this here (which I don't suggest, you're coupling this to a context that it shouldn't need to know about), would centralize these constants somewhere and replace it for a reference to them so that, if you ever want to change this, you don't need to go about doing it everywhere